overviewer_core/textures.py

```diff
@@ -2048,6 +2082,8 @@ def stairs(self, blockid, data):
         164: "assets/minecraft/textures/block/dark_oak_planks.png",
         180: "assets/minecraft/textures/block/red_sandstone.png",
         203: "assets/minecraft/textures/block/purpur_block.png",
+        509: "assets/minecraft/textures/block/crimson_planks.png",
+        510: "assets/minecraft/textures/block/warped_planks.png",
```

Comment:
why are these at 500 as opposed to 1000? not that it really matters, just wondering if there's something you were planning on doing with that

Reply:
These are the block IDs that minecraft uses itself. Only in bedrock these days, but I thought it was best to just continue the existing pattern.